Formula/envoy@1.18.rb

```diff
@@ -1,19 +1,14 @@
-class EnvoyAT117 < Formula
+class EnvoyAT118 < Formula
```

Comment:
Please add this to `formula_renames.json` so that users who have it installed aren't stranded on that version.

Reply:
sure!